recipe/meta.yaml

```diff
@@ -1,9 +1,5 @@
 {% set version = "3.9.0" %}
-{% set build_num = 2 %}
-{% if blas_impl == 'mkl' %}
-{% set version = "3.8.0" %}
-{% set build_num = build_num + 18 %}
-{% endif %}
+{% set build_num = 3 %}
```

Comment:
```suggestion
{% set build_num = 2 %}
```

Reply:
I'm not sure I understand this one - build numbers are cheap, and we're definitely changing something about the mkl builds. Why shouldn't this be bumped?

Reply:
Other builds are not affected right? MKL builds had version 3.8.0 and now it's version 3.9.0

Reply:
> Other builds are not affected right? MKL builds had version 3.8.0 and now it's version 3.9.0

Other builds are not affected, that's true, but isn't the fact that the recipe changes substantially for any build variant enough to bump the build number? At least that's how I've understood it so far (unless your point is that this recipe is only repackaging other dependencies?).